packages/flutter/lib/src/services/text_input.dart

```diff
@@ -1192,4 +1193,62 @@ class TextInput {
       args,
     );
   }
+
+  /// Finishes the current autofill context, and potentially saves the user
+  /// input for future use if [shouldSave] is true.
```

Comment:
```suggestion
  /// input for future use if `shouldSave` is true.
```

Reply:
And elsewhere in this doc since it is a parameter to the method and not a member of the surrounding context that we can link to.

Reply:
It does get rendered differently I think, for instance `children` and `onValueChanged` on this page: https://api.flutter.dev/flutter/cupertino/CupertinoSlidingSegmentedControl/CupertinoSlidingSegmentedControl.html. Also the LSP client I'm using is able to pick up method parameters that are in square brackets.

Reply:
See also: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#dartdoc-specific-requirements